src/main.ts

```diff
@@ -35,6 +35,7 @@ export interface TestArgs {
 	internConfig: string;
 	node: boolean;
 	filter: string;
+	watch?: boolean;
```

Comment:
nit: is this actually optional? The arg has a default so I think it will always be true/false (i don't think it affects anything tho)